src/Workspaces/Core/Portable/Workspace/Solution/CompilationOutputFilePaths.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Paths of files produced by the compilation.
```

Comment:
it's a little odd to me that we have this sturct that indicates it's for multiple file paths.  But we only expose one path, and that path is itself optional.  so we have other paths we think we would expose here in hte future?

Reply:
> so we have other paths we think we would expose here in hte future?

Yes, we expect more paths. See https://github.com/dotnet/roslyn/pull/41745/files/9f60efe2f6f0df7477a0856bca30f88f46e18ba3#diff-74b03a1b69714861b192cbc422ed2660R24-R27